### PR TITLE
Add Problem 120 & 127

### PR DESCRIPTION
**Pull request by AquaIndigo**

### CODE REVIEW

**Review by findmyway — APPROVED** [review]

File: src/problems/120.triangle.jl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#
+# @lc app=leetcode id=120 lang=python3
+#
+# [120] Triangle
+#
+
+# @lc code=start
```
[the remark this answers is not in the document]
Comment (AquaIndigo) [reply src/problems/120.triangle.jl:7]: VScode插件生成的模板比较emmmm

File: src/problems/127.word-ladder.jl [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Output: 0
+
+Explanation: The endWord "cog" is not in wordList, therefore no possible transformation.
+=#
+# import Pkg
+# Pkg.add("DataStructures")
+using DataStructures
+# @lc code=start
+function ladderLength(beginWord::String, endWord::String, wordList::Vector{String})::Int
```
Comment (findmyway) [inline src/problems/127.word-ladder.jl:43]: Let's stick with the BlueStyle to use lower case with underscors for method names. https://github.com/invenia/BlueStyle#synopsis

Comment (findmyway) [inline src/problems/127.word-ladder.jl:43]: Let's stick with the BlueStyle to use lower case with underscors for method names. https://github.com/invenia/BlueStyle#synopsis

File: src/problems/120.triangle.jl [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+]
+The minimum path sum from top to bottom is 11 (i.e., 2 + 3 + 5 + 1 = 11).
+
+Note:
+
+Bonus point if you are able to do this using only O(n) extra space, where n is the total number of rows in the triangle.
+
+=#
+# @lc code=start
+function minimumTotal(triangle::Vector{Vector{Int}})
```
Comment (findmyway) [inline src/problems/120.triangle.jl:25]: Let's stick with the BlueStyle to use lower case with underscors for method names. https://github.com/invenia/BlueStyle#synopsis

**Review by findmyway — APPROVED** [review]

File: src/problems/127.word-ladder.jl [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    if !(endWord in wordList) 
+        return 0
+    end
+    !(beginWord in wordList) && push!(wordList, beginWord)
+    s, t = findall(x -> x == beginWord, wordList)[1], findall(x -> x == endWord, wordList)[1]
+    wl2 = collect.(wordList)
```
[the remark this answers is not in the document]
Comment (AquaIndigo) [reply src/problems/127.word-ladder.jl]: It had unpleasantly influenced  the performance, and I have fixed that.

[the remark this answers is not in the document]
Comment (findmyway) [reply src/problems/127.word-ladder.jl]: 👍

**Pull request merged**